src/multilayer_canopy/MLCanopyFluxesMod.F90

```diff
@@ -362,19 +362,19 @@ subroutine MLCanopyFluxes (bounds, num_exposedvegp, filter_exposedvegp, &
        p = filter_mlcan(fp)
        c = patch%column(p)
        g = patch%gridcell(p)
-       lat = grc%latdeg(g) * pi / 180._r8
-       lon = grc%londeg(g) * pi / 180._r8
+       lat = grc%lat(g)
+       lon = grc%lon(g)
```

Comment:
I did not bring up this one before, either. It's a simplification that does not change answers.